app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -318,6 +326,18 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
     }
   };
 
+  $scope.detectClicked = function($event) {
+    miqService.sparkleOn();
+    miqService.detectWithRest($event, $scope.actionUrl)
+      .then(function success(data) {
+        $scope.$apply(function() {
```

Comment:
..And you can remove this `$scope.$apply` :) (https://github.com/ManageIQ/manageiq-ui-classic/pull/1304/files#r120332564)